client-v2/src/components/Clipboard.tsx

```diff
@@ -40,7 +40,7 @@ const ClipboardWrapper = styled<
   'data-testid': 'clipboard-wrapper'
 })`
   width: ${({ theme, clipboardHasOpenForms }) =>
-    clipboardHasOpenForms ? theme.front.minWidth : 200}px;
+    clipboardHasOpenForms ? theme.front.minWidth : 220}px;
```

Comment:
When the integration tests run and attempt to drag an item from the clipboard to a collection, the area of the item that they select is the item _image_, rather than the item itself. By slightly increasing the clipboard width, the correct area of the item is selected.

Reply:
thanks @jennygrahamjones! This should no longer be a problem in subsequent PRs due to #871.